Model/Quote/Checkout.php

```diff
@@ -305,22 +305,22 @@ private function getDropOffDays(string $carrierPath): array {
     private function getDeliveryOptionsStrings(): array
     {
         return [
-            'deliveryTitle'             => $this->helper->getGeneralConfig('delivery_titles/delivery_title'),
-            'deliveryStandardTitle'     => $this->helper->getGeneralConfig('delivery_titles/standard_delivery_title'),
-            'deliveryMorningTitle'      => $this->helper->getGeneralConfig('delivery_titles/morning_title'),
-            'deliveryEveningTitle'      => $this->helper->getGeneralConfig('delivery_titles/evening_title'),
-            'deliveryPickupTitle'       => $this->helper->getGeneralConfig('delivery_titles/pickup_title'),
-            'pickupTitle'               => $this->helper->getGeneralConfig('delivery_titles/pickup_title'),
-            'deliverySameDayTitle'      => $this->helper->getGeneralConfig('delivery_titles/same_day_title') ?: __('Same day delivery'),
-            'hideSenderTitle'           => $this->helper->getGeneralConfig('delivery_titles/hide_sender_title') ?: __('Hide sender'),
-            'list'                      => $this->helper->getGeneralConfig('delivery_titles/pickup_list_button_title'),
-            'map'                       => $this->helper->getGeneralConfig('delivery_titles/pickup_map_button_title'),
-            'packageTypeMailbox'        => $this->helper->getGeneralConfig('delivery_titles/mailbox_title'),
-            'packageTypeDigitalStamp'   => $this->helper->getGeneralConfig('delivery_titles/digital_stamp_title'),
-            'packageTypePackageSmall'   => $this->helper->getGeneralConfig('delivery_titles/package_small_title') ?: __('Packet'),
-            'signatureTitle'            => $this->helper->getGeneralConfig('delivery_titles/signature_title'),
-            'onlyRecipientTitle'        => $this->helper->getGeneralConfig('delivery_titles/only_recipient_title'),
-            'saturdayDeliveryTitle'     => $this->helper->getGeneralConfig('delivery_titles/saturday_title'),
+            'deliveryTitle'             => $this->helper->getGeneralConfig('delivery_titles/delivery_title') ?: __('delivery_title'),
+            'deliveryStandardTitle'     => $this->helper->getGeneralConfig('delivery_titles/standard_delivery_title') ?: __('standard_delivery'),
+            'deliveryMorningTitle'      => $this->helper->getGeneralConfig('delivery_titles/morning_title') ?: __('morning_title'),
+            'deliveryEveningTitle'      => $this->helper->getGeneralConfig('delivery_titles/evening_title') ?: __('evening_title'),
+            'deliveryPickupTitle'       => $this->helper->getGeneralConfig('delivery_titles/pickup_title') ?: __('pickup_title'),
+            'pickupTitle'               => $this->helper->getGeneralConfig('delivery_titles/pickup_title') ?: __('pickup_title'),
```

Comment:
Klopt het dat deze hetzelfde zijn? (pickup_title)

Reply:
> Klopt het dat deze hetzelfde zijn? (pickup_title)

Nee dit is denk ik een oversight 1 moment